src/magic.h

```diff
@@ -81,6 +82,50 @@ struct fake_spell {
         max_level( max_level ), self( hit_self ) {}
 };
 
+class ter_furn_transform
+{
+
+    private:
+
+        std::string fail_message;
+
+        std::map<ter_id, weighted_int_list<ter_id>> ter_transform;
+        std::map<std::string, weighted_int_list<ter_id>> ter_flag_transform;
+        // the value is message and whether that message is good or bad. defaults to good
```

Comment:
There is no default in a `std::map<ter_id, std::pair<std::string, bool>>`. That "defaults to good" has no meaning for this member. (I assume you refer to the loading from JSON, where it's optional, but why document this one piece of information here?)